pkg/utils/args/percentage.go

```diff
@@ -0,0 +1,48 @@
+package args
+
+import (
+	"fmt"
+	"strconv"
+)
+
+// Percentage implements the flag.Value interface and allows to parse stringified percentages.
+type Percentage struct {
+	val int64
```

Comment:
```suggestion
	val uint64
```

Could also be exported to allow setting a default value, and avoiding the Getter